.wiki/Configuration-Repository.md

```diff
@@ -70,8 +70,8 @@ repo:
 ## File
 
 Files repository is a general purpose file storage which provides API for upload and download: `PUT` requests for upload and `GET` for download.
-To setup this repository, create config with `file` repository type and storage configuration. Permissions configuration can authorize
-users allowed to upload and download.
+To setup this repository, create config with `file` repository type and storage configuration. [Permissions configuration](./Configuration-Repository Permissions.md) 
```

Comment:
The verb 'set up' is spelled as two words. 
The noun 'setup' is spelled as one. 